src/blurb/blurb.py

```diff
@@ -399,7 +399,7 @@ class BlurbError(RuntimeError):
     (This formatting will be inserted when rendering the final output.)
   * Lines longer than 76 characters will be wordwrapped.
       * In the final output, the first line will have
-        "- gh-issue-<gh-issue-number>: " inserted at the front,
+        "- :gh:`<gh-issue-number>`: " inserted at the front,
         and subsequent lines will have two spaces inserted
         at the front.
```

Comment:
Is this referring to files such as [Misc/NEWS.d/3.14.0a4.rst](https://github.com/python/cpython/blob/main/Misc/NEWS.d/3.14.0a4.rst?plain=1)?

If so, none of those BODY sections begin with any kind of issue number.

I don't think this is referring to the merged `Doc/build/NEWS` file? Although it does have lines beginning `- gh-123456:`.

Reply:
I interpreted it as referring to the merged file, and `gh-issue-` just being a typo for `gh-`.

Reply:
Ah sorry, it indeed says "In the _**final**_ output" :)

For example:

```
- gh-129248: The iOS test runner now strips the log prefix from each line
  output by the test suite.
```